js/script.js

```diff
@@ -0,0 +1,67 @@
+const myBtn = document.querySelector('.my-btn');
+
+const eachSpeakers = [
+  {
+    name: 'Ezra Ermy',
+    picture: 'images/ez.JPG',
+    profession: 'Ezra is a full-stack web developer and researcher',
+    backgound: 'Excellent knowledge of JavaScript/TypeScript, HTML and CSS. Provides mentorship and supervise developers',
+  },
+  {
+    name: 'Abenu Ermy',
+    picture: 'images/ez.JPG',
+    profession: 'Ezra is a full-stack web developer and researcher',
+    backgound: 'Excellent knowledge of JavaScript/TypeScript, HTML and CSS. Provides mentorship and supervise developers',
+  },
+  {
+    name: 'Ekut Ermy',
+    picture: 'images/ez.JPG',
+    profession: 'Ezra is a full-stack web developer and researcher',
+    backgound: 'Excellent knowledge of JavaScript/TypeScript, HTML and CSS. Provides mentorship and supervise developers',
```

Comment:
- [ ] According to the Behance template, kindly ensure that by default your application is able to show two speakers on mobile version so when clicked the more button you can see all speakers from the the List 
![Screenshot from 2023-01-06 16-51-05](https://user-images.githubusercontent.com/83042742/218599872-b327211d-233a-4fb0-8fdf-08577ffb6e08.png)